### PR TITLE
fix sticky table header on chrome

### LINKED ISSUE #967: Broken css - Overflowing thead
Selecting ecmascript 5 I get the result below.
screen shot 2016-11-22 at 1 18 03 pm

I'm using 
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2926.0 Safari/537.36

JavaScript: V8 5.6.326
Flash: 24.0.0.154

### PR DESCRIPTION
**Pull request by Trias**
position: sticky is supported on chrome but apparently buggy. so make it use the fallback. Also updated the jquery.floatThead.js dependency

Fixes #967.

### CODE REVIEW

File: master.js [hunk 1]
```diff
@@ -44,14 +44,20 @@ $(function() {
...
+  
+  if (thead.css('position') === "-webkit-sticky") {
     // Remove floatThead when native position:sticky is usable.
     // Currently, only Safari (which floatThead incidentally does not support),
     // and its -webkit-sticky prefix, works correctly with <thead>.
     $.fn.floatThead = function() { return this };
   }
 
+  // position: sticky is implemented in chrome but buggy. currently makes the background transparent sometimes
+  if(thead.css('position') === "sticky" && window.chrome){
```
Comment (ljharb) [inline master.js]: `if (` and `) {`

**Review by ljharb — CHANGES_REQUESTED** [review]

**Review by ljharb — APPROVED** [review]
LGTM but it'd be great to rebase this down to fewer commits

**Review by chicoxyzzy — APPROVED** [review]
LGTM

**Comment by Trias** [comment]
ok done. hope it's fine now :)

**Comment by chicoxyzzy** [comment]
Thank you!

**Review by mkoryak — COMMENTED** [review]
Cool. I didn't know safari (of all browsers) now supports sticky. I'll have to pull this into floatthead

**Pull request merged**